### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by vpodzime**
Documentation for structs and a few missing items added to the right places.

### CODE REVIEW

**Comment by vpodzime** [comment]
The tests will probably fail due to some changes in the new kernel (more specifically in the *zram* interface) installed to the jenkins slaves.

File: src/lib/plugin_apis/mdraid.api [hunk 1]
```diff
@@ -20,6 +20,20 @@ typedef enum {
...
+/**
+ * BDMDExamineData:
+ * @device: path of the MD device
+ * @level: RAID level of the device
+ * @num_devices: number of devices used by the MD device
+ * @name: name of the MD device
+ * @size: size of the MD device
+ * @uuid: array UUID
+ * @update_time: update time of the MD device
+ * @dev_uuid: UUID of the MD device
```
Comment (dwlehman) [inline src/lib/plugin_apis/mdraid.api]: You could say something like `UUID of the member device` here to make it a bit clearer.

**Comment by dwlehman** [comment]
Looks good to me other than the one possible language refinement I proposed.

File: src/lib/plugin_apis/mdraid.api (hunk 1 above)
Comment (vpodzime) [inline src/lib/plugin_apis/mdraid.api]: Good point, thanks!

**Pull request merged**